powerstore/resource_volume_group_snapshot_test.go

```diff
@@ -178,15 +240,33 @@ resource "powerstore_volumegroup_snapshot" "test" {
   name = "test_snap"
   description = "Test Snapshot Resource"
   volume_group_id="` + volumeGroupID + `"
-  expiration_timestamp="2023-05-06T09:01:47Z"
+  expiration_timestamp="2035-05-06T09:01:47Z"
+}
+`
+
+var VolumeGroupSnapParamInvalidVolumeID = `
+resource "powerstore_volumegroup_snapshot" "test" {
+  name = "test_snap"
+  description = "Test Snapshot Resource"
+  volume_group_id="5c3c103a-9373-4f50-a34a"
```

Comment:
Better to create the prerequisites on the fly so that even if the cluster changes, there will not be an issue in running tests.
If its a big effort, can be taken in separate ticket.